worlds/ahit/__init__.py

```diff
@@ -0,0 +1,356 @@
+from BaseClasses import Item, ItemClassification, Tutorial, Location
+from .Items import item_table, create_item, relic_groups, act_contracts, create_itempool
+from .Regions import create_regions, randomize_act_entrances, chapter_act_info, create_events, get_shuffled_region
+from .Locations import location_table, contract_locations, is_location_valid, get_location_names, TASKSANITY_START_ID, \
+    get_total_locations
+from .Rules import set_rules
+from .Options import AHITOptions, slot_data_options, adjust_options
+from .Types import HatType, ChapterIndex, HatInTimeItem
+from .DeathWishLocations import create_dw_regions, dw_classes, death_wishes
+from .DeathWishRules import set_dw_rules, create_enemy_events
+from worlds.AutoWorld import World, WebWorld
+from typing import List, Dict, TextIO
+from worlds.LauncherComponents import Component, components, icon_paths, launch_subprocess, Type
+from Utils import local_path
+
+
+def launch_client():
+    from .Client import launch
+    launch_subprocess(launch, name="AHITClient")
+
+
+components.append(Component("A Hat in Time Client", "AHITClient", func=launch_client,
+                            component_type=Type.CLIENT, icon='yatta'))
+
+icon_paths['yatta'] = local_path('data', 'yatta.png')
+
+hat_craft_order: Dict[int, List[HatType]] = {}
+hat_yarn_costs: Dict[int, Dict[HatType, int]] = {}
+chapter_timepiece_costs: Dict[int, Dict[ChapterIndex, int]] = {}
+excluded_dws: Dict[int, List[str]] = {}
+excluded_bonuses: Dict[int, List[str]] = {}
+dw_shuffle: Dict[int, List[str]] = {}
+nyakuza_thug_items: Dict[int, Dict[str, int]] = {}
+badge_seller_count: Dict[int, int] = {}
```

Comment:
This is what I wanted to do, but I ran into some strange problems where it would copy data from one AHIT world to another AHIT world in the same seed, so I just made them separate

Reply:
That sounds like they might've been _class_ variables instead of instance variables? (are you familiar with that gotcha?)

Reply:
```py
class HatStuff:
    hat1 = [] # this is a class variable, shared by all instances of the class ("static" in some languages)
    
    def __init__():
        self.hat2 = [] # this is an instance variable, every instance object gets its own separate value
```

Reply:
Done